rwx/src/main/java/org/commonjava/rwx/vocab/Nil.java

```diff
@@ -15,16 +15,28 @@
  */
 package org.commonjava.rwx.vocab;
 
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.io.Serializable;
+
 /**
  * Created by ruhan on 12/14/17.
  */
 public class Nil
+    implements Serializable
 {
-    public static Nil NIL_VALUE = new Nil();
+    private static final long serialVersionUID = 5398616344106741077L;
 
-    private Nil()
-    {
-    }
+    private static final int VERSION = 1;
```

Comment:
This was added in order to track the format version for Externalizable, akin to serialVersionUID for Serializable.